Header header
MMetaData info
float32[] velocities
float32[] data